tests/sqllogictests/suites/query/02_function/02_0062_function_unnest

```diff
@@ -369,6 +369,35 @@ drop table t;
 statement ok
 drop table t2;
 
+statement ok
+drop table if exists fold_count;
+
+statement ok
+create table fold_count(id array(int));
+
+statement ok
+insert into fold_count values([1,2,3,4])
+
+query T
+select concat(c,'y') as b from (select concat(c::String, 'x')  as c from (select unnest(id) as c from fold_count));
+----
+1xy
+2xy
+3xy
+4xy
+
+query I
+select count(b) from (select concat(c,'y') as b from (select concat(c::String, 'x')  as c from (select unnest(id) as c from fold_count)));
+----
+4
+
+#ERROR 1105 (HY000): Code: 1006, Text = Unable to get field named "0". Valid fields: [].
+statement error 1006
+select count() from (select unnest(id) from fold_count)
```

Comment:
It's error of unnest ?

Reply:
the main build returns `1`, which is not a error but still incorrect.